config/src/main/java/org/hyperledger/besu/config/MergeConfigOptions.java

```diff
@@ -18,6 +18,7 @@
 
 public class MergeConfigOptions {
   private static final AtomicBoolean mergeEnabled = new AtomicBoolean(false);
+  private static final AtomicBoolean rollupExtensionEnabled = new AtomicBoolean(false);
```

Comment:
Extending besu for rollup support likely warrants its own configuration class. You probably don't want to put it here since this should be getting cleaned up soon now that we are post-merge.

Reply:
This change was actually committed by mistake, as it does it is not used in the remaining changes. I will revert it.
Will keep in mind this class will be removed in the future and create rollup own config options.